glambie/data/timeseries.py

```diff
@@ -12,21 +12,30 @@
 @dataclass
 class TimeseriesData():
     """Class to wrap the actual data contents of a Timeseries"""
```

Comment:
I think we're now talking about a timeseries that is not continuous / has gaps?
I recommend explaining this, as it's not instantly obvious how a timeseries could have multiple starting dates.